source/HailPanel.cpp

```diff
@@ -72,7 +72,7 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 	else if(ship->IsDisabled())
 	{
 		const Ship *flagship = player.Flagship();
-		if(!flagship->JumpsRemaining() || flagship->IsDisabled())
```

Comment:
Why this change?

Reply:
That might be from when I was trying to make evasive ships mute... I'll fix it